app/component/control.js

```diff
@@ -34,7 +34,41 @@ export default class Control extends Component {
             </Row>
             <Row>
                 <Button
-                    onPress={this.props.onMissx3}
+                    onPress={this.props.onHitAllThisRound}
+                    title={"Hit all"}
+                    accessibilityLabel={"Hit all"}
```

Comment:
What happens if the user presses "Miss 1" and then presses "Hit all"? If it's "darts 2 and 3 are marked as hit" then maybe the button should be "Hit rest"? (To match Miss rest?)

Edit: oh, found the behavior I think - it only works if the round is new. But I'm still kinda confused then. Should "miss rest" actually be "miss all" for consistency? Or was it intentional to make these non-symmetric?

Reply:
yep. done